src/backend/ai-reviewer-api/src/main/java/ca/bc/gov/open/jag/aireviewerapi/document/DocumentsApiDelegateImpl.java

```diff
@@ -98,7 +100,11 @@ public DocumentsApiDelegateImpl(
         //Due to a bug in mapstruct logic has been performed inline
         ExtractRequest extractRequest = extractRequestMapper.toExtractRequest(extractRequestMapper.toExtract(xTransactionId, (xUseWebhook == null || xUseWebhook)), xDocumentType, file, receivedTimeMillis);
 
+        logger.info("document submission to diligen start, filename: " + file.getName());
+        stopWatch.start();
         BigDecimal response = diligenService.postDocument(xDocumentType, file, documentTypeConfiguration.getProjectId());
+        stopWatch.stop();
+        logger.info("document submission to diligen finish, filename: " + file.getName() + ", size: " + file.getSize() + " bytes, time: " + stopWatch.getLastTaskTimeMillis() + " ms");        
```

Comment:
Nice this covers it. I think we want to move it to handleDocumentEvent. There is a total processing time in the EventRequest Object. We may want to use transaction id as well. My only concern with filename it we don't control it. We may want to ask Jaime if there is ever a persons name in it.

Reply:
Sounds good.  On it. I was wondering about the filename myself ... don't know if it's considered PI

Reply:
Moved to handleDocumentEvent and simply enhanced the logging there to also include transactionID.